content-src/components/Spotlight/Spotlight.js

```diff
@@ -1,11 +1,16 @@
 const React = require("react");
+const DEFAULT_LENGTH = 3;
+const SiteIcon = require("components/SiteIcon/SiteIcon");
 
 const SpotlightItem = React.createClass({
   render() {
     const site = this.props;
+    const imageUrl = site.images[0].url;
+    const iconUrl = site.favicon_url;
     return (<li className="spotlight-item">
-      <div className="spotlight-image" style={{backgroundImage: `url(${site.image})`}} ref="image">
-        <div className="spotlight-icon" style={{backgroundImage: `url(${site.icon})`}} ref="icon" />
+      <div className="spotlight-image" style={{backgroundImage: `url(${imageUrl})`}} ref="image">
```

Comment:
at the moment i don't think so, since it needs to be resized to cover the element with `cover`